packages/gestalt/src/Button.js

```diff
@@ -290,6 +293,8 @@ ButtonWithForwardRef.propTypes = {
   inline: PropTypes.bool,
   name: PropTypes.string,
   onClick: PropTypes.func,
+  // eslint-disable-next-line react/forbid-prop-types
+  onNavigationOptions: PropTypes.object,
```

Comment:
It looks like we can type this more specifically? At the very least we can define the type once (where the flow type is defined) and only suppress once, there

Reply:
DONE